src/main/kotlin/org/b12x/gfe/core/model/parsers/ParserHlaData.kt

```diff
@@ -0,0 +1,26 @@
+package org.b12x.gfe.core.model.parsers
+
+import com.beust.klaxon.JsonArray
+import com.beust.klaxon.Parser
+import okhttp3.Response
+import org.b12x.gfe.core.model.DataDownload
+
+object ParserHlaData {
+
+    const val DB_HLA = "http://gfe.b12x.org/v1/imgt-versions"
+
+    val hlaDataDownload = DataDownload("hla")
+
+    fun parseResponse(response: Response): JsonArray<String> {
+        val body = response.body?.string() ?: ""
+        val stringBuilder: StringBuilder = StringBuilder(body)
```

Comment:
There's no need to explicitly specify the type of a variable or constant when immediately initializing it. Ditto on next line.